components/VotePanel/YouVoted.tsx

```diff
@@ -179,7 +179,7 @@ export const YouVoted = ({ quorum }: { quorum: 'electoral' | 'veto' }) => {
               onClick={() => submitRelinquishVote()}
               disabled={!isWithdrawEnabled || isLoading}
             >
-              Withdraw
+              Relinquish Vote
```

Comment:
Wouldn't be surprised if this causes awkward linebreaks at certain window widths, but if you dont think so I'll take your word for it.

Reply:
don't think it does. Ran through all devices in the chrome inspector and looked like it had ample room